source/usage-examples/findOne.go.result

```diff
@@ -0,0 +1 @@
+map[imdb:map[id:368226 rating:3.5 votes:25673] title:The Room
```

Comment:
The benefit of having code output in a separate file is that we can `>` pipe the output of `findOne.go` to this file when we run it. Just helps keep code example output up to date. The reason I chose the extension "<program name>.<lang extension>.result" was to avoid naming collisions. Naming the output "findOne.txt" would collide with the "findOne.txt" rst page. Would you prefer "findOne.go.txt"?

I'll change this to "findOne.go.txt" in the meantime. Let me know if you disagree with my reasoning for keeping this in a separate file. Fix:

Reply:
I don't mind keeping it in a separate file.

At current, it's not an example we're thinking of changing often, so it could be overcomplicating it a bit (some call this https://martinfowler.com/bliki/Yagni.html). Let's keep things simple when we can.

I also don't think what you're proposing is of any benefit since piping output has nothing to do with the example itself. You could also pipe the output of the example to `sed` to replace specific lines directly in the main file if you wanted.

Reply:
A few suggestions if you think namespace collisions are a potential issue:
- create a new directory for output files that you include in the examples
- add something in the filename that indicates they're output, but use the txt extension at the end

Reply:
Thanks for the follow up. I have renamed the output file "findOne-output.txt". I agree that my thinking  around piping output is more likely to complicate the repo than make it easier to maintain. Fix: https://github.com/mongodb/docs-golang/pull/2/commits/b20ac6df71039f1146dd56ecfbcd8964a21b2998